pkg/ovs/ovn-nb-dhcp_options.go

```diff
@@ -157,7 +170,16 @@ func (c *OVNNbClient) updateDHCPv6Options(lsName, cidr, options string) (uuid st
 	/* update */
 	if dhcpOpt != nil {
 		dhcpOpt.Cidr = cidr
-		dhcpOpt.Options = parseDHCPOptions(options)
+		newOptions := parseDHCPOptions(options)
+		// Append new options to existing options
```

Comment:
```suggestion
		// append necessary options to existing options
```

Reply:
I think it should be "append necessary options to new options" or "append new options to existing options"